doc/user_guide/signal1d.rst

```diff
@@ -12,32 +12,32 @@ signals in the Signal1D class.
 Cropping
 --------
 
-The :py:meth:`~._signals.signal1d.Signal1D.crop_signal1D` crops the
+The :py:meth:`~._signals.signal1d.Signal1D.crop_signal` crops the
 spectral energy range *in-place*. If no parameter is passed, a user interface
 appears in which to crop the one dimensional signal. For example:
 
 .. code-block:: python
 
-    s = hs.datasets.example_signals.EDS_TEM_Spectrum()
-    s.crop_signal1D(5, 15) # s is cropped in place
+    >>> s = hs.datasets.example_signals.EDS_TEM_Spectrum()
+    >>> s.crop_signal(5, 15) # s is cropped in place
 
 Additionally, cropping in HyperSpy can be performed using the :ref:`Signal
 indexing <signal.indexing>` syntax. For example, the following crops a spectrum
 to the 5 keV-15 keV region:
 
 .. code-block:: python
 
-    s = hs.datasets.example_signals.EDS_TEM_Spectrum()
-    sc = s.isig[5.:15.] # s is not cropped, sc is a "cropped view" of s
+    >>> s = hs.datasets.example_signals.EDS_TEM_Spectrum()
```

Comment:
Does this sample dataset move to `exspy`?

(loaded several times in this file)